### PR TITLE
[CAM8432] add the new section that contains structure and example

### PR DESCRIPTION
**Pull request by aryaroudi**

### CODE REVIEW

**Comment by aryaroudi** [comment]
Hi @gsanchezu please review this pull request.

Thanks

File: CAM Platform/organizations-api.md [hunk 1]
```diff
@@ -3,40 +3,58 @@
...
+| Parameter | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name | string | the name of the organization | Yes |
+
+### Response
+#### Normal Response Codes
+- **200** Accepted
+
+#### Common Error Response Codes
+- Unauthorized (401) - Invalid access token/cookie
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 401: Unauthorized - Invalid access token/cookie
```

File: CAM Platform/organizations-api.md [hunk 2]
```diff
@@ -3,40 +3,58 @@
...
+- User doesn’t belong to the organization (403)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 403: Forbidden - User doesn’t belong to the organization
```

File: CAM Platform/organizations-api.md [hunk 3]
```diff
@@ -3,40 +3,58 @@
...
+- Not Found (404)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 404: Not Found - No organization found with the given name
```

File: CAM Platform/organizations-api.md [hunk 4]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+            "description": "Manage cloud hosting, Linux and Windows machines",
+            "pricing": []
+        },
+        {
+            "enabled": true,
+            "type": "CenturyLink DCC Foundation",
+            "description": "Manage cloud hosting, Linux and Windows machines.",
+            "pricing": []
+        }
+    ],
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]: Providers collection is an important one, so please add all returned providers.

File: CAM Platform/organizations-api.md [hunk 5]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+    "account_installed_product": "******",
+    "clc_alias": "*****",
+    "plugins": [],
+    "account_status": "Active",
+    "vantive_id": "S626423",
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
    "vantive_id": "*******",
```

File: CAM Platform/organizations-api.md [hunk 6]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+        },
+        {
+            "type": "workspace",
+            "role": "administrator",
+            "id": "**************"
+        },
+        {
+            "role": "administrator",
+            "type": "ldap",
+            "id": "CN=Cloud Application Manager Support,OU=Internal Server Security Groups,OU=Protected,OU=EmployeeAccts,OU=ManagedHosting,DC=na,DC=msmps,DC=net"
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
            "id": "CN=Cloud Application Manager Support,OU=Internal Server Security Groups,OU=******,OU=******,OU=ManagedHosting,DC=na,DC=msmps,DC=net"
```

File: CAM Platform/organizations-api.md [hunk 7]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+        }
+    ],
+    "icon": "/services/blobs/download/58888fc42364025f58d42b9a/mission-field.png",
+    "federated_to": [
+        "elasticbox",
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion

```

File: CAM Platform/organizations-api.md [hunk 8]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+        "centurylink",
+        "contoso",
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion

```

File: CAM Platform/organizations-api.md [hunk 9]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
+
+### Response
+#### Normal Code
+- **200** accepted
+
+#### Error Codes
+- Unauthorized (401) - Invalid access token/cookie
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]: Please, put error codes before description in **ALL**:
```suggestion
- 401: Unauthorized - Invalid access token/cookie
```

File: CAM Platform/organizations-api.md [hunk 10]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+- User doesn’t belong to the organization (403)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 403: Forbidden - User doesn’t belong to the organization
```

File: CAM Platform/organizations-api.md [hunk 11]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+- Not Found (404)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 404 - Not Found
```

File: CAM Platform/organizations-api.md [hunk 12]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
+
+### Response
+#### Normal Code
+- **200** accepted
+
+#### Error Codes
+- Unauthorized (401) - Invalid access token/cookie
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 401: Unauthorized - Invalid access token/cookie
```

File: CAM Platform/organizations-api.md [hunk 13]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+- User doesn’t belong to the organization (403)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 403: Forbidden - User doesn’t belong to the organization
```

File: CAM Platform/organizations-api.md [hunk 14]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+- Not Found (404)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 404: Not Found
```

File: CAM Platform/organizations-api.md [hunk 15]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
-      {
-         "name":"user",
-         "type":"User ID",
-         "value":null
+### Response
+#### Normal Code
+- **200** accepted
+
+#### Error Codes
+- Unauthorized (401) - Invalid access token/cookie
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 401: Unauthorized - Invalid access token/cookie
```

File: CAM Platform/organizations-api.md [hunk 16]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+- User doesn’t belong to the organization (403)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 403: Forbidden - User doesn’t belong to the organization
```

File: CAM Platform/organizations-api.md [hunk 17]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+- Not Found (404)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]:
```suggestion
- 404: Not Found
```
Please, review **ALL** Response > Error Codes and put code before description

File: CAM Platform/organizations-api.md [hunk 18]
```diff
@@ -56,251 +74,678 @@ ElasticBox-Release: 4.0
...
+| Organization name |  String  |  The name of the organization | Yes |
 
-Updates an existing organization given its name. Only the organization administrator can update.
+### Response
+#### Normal Code
+- **200** accepted
 
-**Normal Response Codes**
+#### Error Codes
+- Unauthorized (401) - Invalid access token/cookie
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]: Same as above

File: CAM Platform/organizations-api.md [hunk 19]
```diff
@@ -543,8 +884,18 @@ ElasticBox-Release: 4.0
...
 
-**Response Parameters**
+### Response
+#### Normal Code
+- **200** accepted
 
+#### Error Codes
+- Bad Request (400) - Request missing, incomplete or includes invalid properties (details provided inside body)
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]: Same as above for ALL

File: CAM Platform/organizations-api.md [hunk 20]
```diff
@@ -791,25 +1038,35 @@ ElasticBox-Release: 4.0
...
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
+
+### Response
+#### Normal Code
+- **202** accepted
+
+#### Error Codes
+- Unauthorized (401) - Invalid access token/cookie
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]: Same as above

**Comment by gsanchezu** [comment]
Hey @jim-phillips, this is ready to merge.

File: CAM Platform/organizations-api.md [hunk 21]
```diff
@@ -1,42 +1,60 @@
...
 }}}
 
 ### Manage an Organization
 
 | Resource | Description |
 |----------|-------------|
-| GET /services/organizations/{organization_name} | Gets the schema of the given organization. |
-| PUT /services/organizations/{organization_name} | Updates an existing organization. |
-| PUT /organizations/{organization_name}/sync_groups | Queues a request to sync LDAP groups. |
+| [GET /services/organizations/{organization_name}](#get-servicesorganizationsorganization_name) | Get the schema of the given organization. |
```
Comment (gsanchezu) [inline CAM Platform/organizations-api.md]: Hey @aryaroudi, I realized this one has not been updated to include links in the summary table. Could you please update? Thanks!

**Comment by gsanchezu** [comment]
Hey @robersantamaria, could you please review this one? Thanks!

File: CAM Platform/organizations-api.md [hunk 22]
```diff
@@ -577,22 +997,7 @@ ElasticBox-Release: 4.0
    "administrators":[
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: Same as above.

File: CAM Platform/organizations-api.md [hunk 23]
```diff
@@ -732,31 +1074,6 @@ ElasticBox-Release: 4.0
...
-         "type":"Service Instance ID",
-         "value":null
-      },
-      {
-         "name":"Testing",
-         "type":"Custom",
-         "value":"test"
       }
    ],
    "cost_centers":[
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: I don't think this is used anymore.

File: CAM Platform/organizations-api.md [hunk 24]
```diff
@@ -787,29 +1104,39 @@ ElasticBox-Release: 4.0
...
+#### Headers
 ```
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
+#### URI Parameters
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: `organization_name`

File: CAM Platform/organizations-api.md [hunk 25]
```diff
@@ -787,29 +1104,39 @@ ElasticBox-Release: 4.0
...
+### Response
+#### Normal Code
+- **202** accepted
 
+#### Error Codes
+- 401: Unauthorized - Invalid access token/cookie
+- 404: Not Found
+
+
+#### Response Parameters
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: There is no body in the response to `sync_groups`

File: CAM Platform/organizations-api.md [hunk 26]
```diff
@@ -828,7 +1155,7 @@ ElasticBox-Release: 4.0
...
 
-**Response Body**
+#### Response Body
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: Same as above.

File: CAM Platform/organizations-api.md [hunk 27]
```diff
@@ -1,42 +1,60 @@
...
 
-**Response Parameters**
+#### URI Parameters
+| Parameter | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name | string | the name of the organization | Yes |
+
+### Response
+#### Normal Response Codes
+- **200** Accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **200** OK

File: CAM Platform/organizations-api.md [hunk 28]
```diff
@@ -56,251 +74,748 @@ ElasticBox-Release: 4.0
...
-         "value":null
+#### URI Parameters
+
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
+
+### Response
+#### Normal Code
+- **200** accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **200** OK

File: CAM Platform/organizations-api.md [hunk 29]
```diff
@@ -56,251 +74,748 @@ ElasticBox-Release: 4.0
...
+```
+
+#### URI Parameters
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
+
+### Response
+#### Normal Code
+- **200** accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **200** OK

File: CAM Platform/organizations-api.md [hunk 30]
```diff
@@ -56,251 +74,748 @@ ElasticBox-Release: 4.0
...
-### PUT /services/organizations/{organization_name}
+#### URI Parameters
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
 
-Updates an existing organization given its name. Only the organization administrator can update.
+### Response
+#### Normal Code
+- **200** accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **200** OK

File: CAM Platform/organizations-api.md [hunk 31]
```diff
@@ -543,8 +954,18 @@ ElasticBox-Release: 4.0
...
 
-**Response Parameters**
+### Response
+#### Normal Code
+- **200** accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **200** OK

File: CAM Platform/organizations-api.md [hunk 32]
```diff
@@ -56,251 +74,748 @@ ElasticBox-Release: 4.0
...
+```
+
+#### URI Parameters
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
+
+### Response
+#### Normal Code
+- **200** accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **200** OK

File: CAM Platform/organizations-api.md [hunk 33]
```diff
@@ -787,29 +1104,39 @@ ElasticBox-Release: 4.0
...
 ```
+#### URI Parameters
+| Name | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Organization name |  String  |  The name of the organization | Yes |
 
-**Response Parameters**
+### Response
+#### Normal Code
+- **202** accepted
```
Comment (robersantamaria) [inline CAM Platform/organizations-api.md]: **202** Accepted

**Comment by aryaroudi** [comment]
Hi @robersantamaria Please review the changes.

Thanks

**Pull request merged**